templates/ZtrBoardGame.Console.Tests/StepDefinitions/BoardConnectivityStepDefinitions.cs

```diff
@@ -108,6 +108,48 @@ public void ThenAnErrorMessageShouldBeDisplayedInTheConsole(string errorMessage)
         _console.Output.Should().Contain(errorMessage);
     }
 
+    [Given(@"a board is configured with the PC server address")]
```

Comment:
@jules, I am afraid if this file isn't too big mess. Can we organize it with `#region` and things like that to make possible easiest navigation on this file in  copmarisong to the feature file?